src/nvofapi/nvofapi_d3d12_instance.cpp

```diff
@@ -44,8 +45,9 @@ namespace dxvk {
     }
 
     bool NvOFInstanceD3D12::Initialize() {
-        m_library = LoadLibraryA("winevulkan.dll");
-        if (!m_library) {
+        m_vk = m_resourceFactory.CreateVulkan("winevulkan.dll");
```

Comment:
Technically this changes the behavior from loading the library into just retrieving a handle of already-loaded one. Probably doesn't matter though, it shouldn't be possible to create `ID3D12Device` from vkd3d-proton without having winevulkan loaded.

Reply:
Thanks, I've added this to the commit message.